src/EnergyPlus/Furnaces.cc

```diff
@@ -921,6 +942,11 @@ namespace Furnaces {
 		}
 		CheckEquipName.dimension( NumFurnaces, true );
 
+		for (IHPCoilIndex = 1; IHPCoilIndex <= NumFurnaces; ++IHPCoilIndex)//clear IHP flag
+			Furnace(IHPCoilIndex).bIsIHP = false;
```

Comment:
Is this needed? bIsIHP is already set to false in the header so no need to do it again.